joy2twist/include/joy2twist/joy2twist_node.hpp

```diff
@@ -13,6 +13,7 @@
 #include <sensor_msgs/msg/joy.hpp>
 #include <std_msgs/msg/bool.hpp>
 #include <std_srvs/srv/trigger.hpp>
+#include "diagnostic_updater/diagnostic_updater.hpp"
```

Comment:
```suggestion
#include <diagnostic_updater/diagnostic_updater.hpp>
```
Should be grouped with rclcpp include, and inside chevrons